internal/learn/about-iota/an-introduction-to-iota.md

```diff
@@ -1,74 +1,15 @@
 ---
 id: an-introduction-to-iota
-title: An Introduction to IOTA
-description: A basic introduction to the IOTA DLT Technology. The use-cases and
-  the Ecosystem developed around it.
+title: IOTA
+description: A basic introduction to the IOTA protocol.
```

Comment:
```suggestion
description: IOTA runs on the Tangle, a structure where newer transactions verify the older ones, and pretty much any other DLT runs on a blockchain instead.
```
We should also add an image tag to these files  so there's a nice thumbnail if they're shared :)   Off the top of my head a screenshot from the gifs could work, or a nice IOTA logo.